news/2018-02-20-community-choice-2017-results.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Community Choice 2017"
+date: 2018-02-20 08:00:00 +0000
+---
+
+The results are finally in, and it is time to announce the Community Favourites of 2017!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+After some considerable deliberation as to how to handle the results, we've decided to go with a simple tally of most votes received to determine the winner for this year's vote, in a departure from the tradition of weighting the votes based on a variety of factors.
+
+Given that the vote this year is purely aimed at finding the most popular map from the community among each mode, anything more than that would be unwarranted!
+
+Without further ado, here are the four grand winners from each mode:
+
+## osu!
+
+![](/wiki/shared/news/2018-02-20-community-choice-2017-results/hitorigoto.jpg)
+
+In an overwhelming display of support that should surprise exactly nobody, [Doormat](https://osu.ppy.sh/u/3230571)'s set of [ClariS - Hitorigoto -TV MIX-](https://osu.ppy.sh/s/596704) takes the title for Community Favourite of 2017 with an outstanding **4434** votes, nearly a full thousand more than its nearest competitor.
```

Comment:
Should include "for osu!" or "for the osu! game mode" after "...the title for Community Favourite of 2017..." to clarify this section is about that game mode, as you did for the other modes later on